pkg/controllers/repositoryserver/utils.go

```diff
@@ -225,3 +229,22 @@ func WaitTillCommandSucceed(ctx context.Context, cli kubernetes.Interface, cmd [
 	})
 	return err
 }
+
+func getVolumes(ctx context.Context, cli kubernetes.Interface, secret *corev1.Secret, namespace string) (map[string]string, error) {
+	vols := make(map[string]string, 0)
+	var claimName []byte
+	if len(secret.Data) == 0 {
+		return nil, errors.Errorf(secerrors.EmptySecretErrorMessage, secret.Namespace, secret.Name)
+	}
+	if locationType, ok := (secret.Data[reposerver.TypeKey]); ok && reposerver.LocType(string(locationType)) == reposerver.LocTypeFilestore {
+		if claimName, ok = secret.Data[reposerver.ClaimNameKey]; !ok {
+			return nil, errors.New("Claim name not set for file store location secret, failed to retrieve PVC")
+		}
+		claimNameString := string(claimName)
+		if _, err := cli.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, string(claimNameString), metav1.GetOptions{}); err != nil {
```

Comment:
```suggestion
		if _, err := cli.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, claimNameString, metav1.GetOptions{}); err != nil {
```
Its already in the string type, converted on the live above